lib/handlers/pools/pool-caching/v2/v2-dynamo-cache.ts

```diff
@@ -18,6 +18,19 @@ export class V2DynamoCache implements ICache<{ pair: Pair; block?: number }> {
       },
     })
   }
+
+  async batchGet(keys: Set<string>): Promise<Record<string, { pair: Pair; block?: number | undefined } | undefined>> {
```

Comment:
fwiw, i dont see this function used anywhere?

Reply:
No it's not being used, hence my comment in https://github.com/Uniswap/routing-api/pull/362/files#r1323540387. @mikeki does favor using it at some point, which I agree.

Reply:
i would add a todo over it so people know what its about

Reply:
Added TODO with ROUTE-81 & ROUTE-84 so that we can triage